lib/money/money.rb

```diff
@@ -207,11 +207,20 @@ def to_d
   end
 
   def to_s(style = nil)
-    case style
+    units = case style
     when :legacy_dollars
-      sprintf("%.2f", value)
+      2
     when :amount, nil
-      sprintf("%.#{currency.minor_units}f", value)
+      currency.minor_units
+    else
+      return nil
```

Comment:
>CI seems to fail not because of tests, but because of coverage collection.

I believe this line of code is not tested. Code coverage needs to be 100% on this repo

Reply:
This line of code existed before (case/switch statement without an `else` clause), but depending on discussion in https://github.com/Shopify/money/pull/212#discussion_r504044321, i'm happy to add a test that asserts an unknown `style` returns `nil`.

Reply:
Looks like we were missing a test, probably best to make it raise an argument error if it's not something we expect